src/i18n/strings/en_EN.json

```diff
@@ -726,6 +726,21 @@
     "Unknown App": "Unknown App",
     "This homeserver is not configured to display maps.": "This homeserver is not configured to display maps.",
     "This homeserver is not configured correctly to display maps, or the configured map server may be unreachable.": "This homeserver is not configured correctly to display maps, or the configured map server may be unreachable.",
+    "Toggle attribution": "Toggle attribution",
+    "Map feedback": "Map feedback",
+    "Enter fullscreen": "Enter fullscreen",
+    "Exit fullscreen": "Exit fullscreen",
+    "Find my location": "Find my location",
+    "Location not available": "Location not available",
+    "Mapbox logo": "Mapbox logo",
+    "Reset bearing to north": "Reset bearing to north",
+    "Zoom in": "Zoom in",
+    "Zoom out": "Zoom out",
+    "ft": "ft",
```

Comment:
Is there a way to add annotations to this so translators can identify what they are?

Reply:
Not that I know of, I was considering just excluding them. I'm not sure we use the maplibre features that expose these strings anyway.

Reply:
Excluding them seems fine.